README.md

```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
+English is widely spoken everywhere. You might hear [Singlish](https://www.aussiepete.com/2008/05/singlish-language-guide-for-foreigners.html) from time to time among Singaporeans. It's fun to learn, but you will sound silly if you try to imitate it.
 
-[Comment: You'll look like an idiot if you try this slang. I sometimes answer
-"are you able to?" questions with "can!", but mostly that just makes me look
-like an idiot. YMMV.]
+## Shopping
 
-## What to buy to bring back home
+Here are a few food suggestions to bring back home:
 
-- Bak Kwa
-  For those who like their pork. Just walk into any
-  [Bee Cheng Hiang](http://www.beechenghiang.com.sg/our-story/our-bakkwa.html) ([outlets at Changi Airport](http://www.changiairport.com/shopping-and-dining/shopping/bee-cheng-hiang.html)) or [Lim Chee Guan](http://www.limcheeguan.com.sg/)
-- Pandan Cake or Kueh Lapis from Bengawan Solo
-  Conveniently they too, have [outlets at Changi Airport](http://www.changiairport.com/shopping-and-dining/shopping/bengawan-solo).
-- Pineapple cakes from [Sunny Hills](http://www.sunnyhills.com.sg/stores.php)
+- [Bak Kwa](https://en.wikipedia.org/wiki/Bakkwa), a Chinese salty-sweet dried meat product similar to jerky. 
+  [Bee Cheng Hiang](http://www.beechenghiang.com.sg/our-story/our-bakkwa.html) or [Lim Chee Guan](http://www.limcheeguan.com.sg/) are both good options.
+- [Pandan Cake](https://en.wikipedia.org/wiki/Pandan_cake) or [Kueh Lapis](https://en.wikipedia.org/wiki/Kue_lapis) from [Bengawan Solo](https://bengawansolo.sg)
+- Pineapple cakes from [Sunny Hills](https://www.sunnyhills.com.sg/collections/all-pineapple-cakes-product)
 - Kaya Jam and Kopi powder from [Ya Kun](http://yakun.com/menu/merchandise)
-- Bak Kut Teh stock powder from [Song Fa](http://www.songfa.com.sg/bak-kut-teh-spice.html)
+- Bak Kut Teh products from [Song Fa](https://songfa.com.sg/pages/retail-products)
 
 ## Don'ts
 
+Avoid these while in Singapore:
+
 - Vandalism
 - Littering
+- Drugs
 - Chewing gum
-  [Comment: Actually you can chew gum like a cow chews cud, but you won't be
-  able to buy any gum in Singapore and when you need to get rid of your gum,
-  make sure you put it in a bin - see "littering" above.]
-- Don't carry [Durian](http://en.wikipedia.org/wiki/Durian) in public transport
-  [Comment: Eating Durian is another must-try experience in Singapore. However, carrying them in public transport is an offence and you will be fined if caught. If you really want to carry it onto public transport, request for them to be packaged on the spot or consider purchasing ones that have already been packed in styrofoam packages wrapped in clingfilm.]
-- Drink and eat in public transport. You can be fined for it. This is especially true for MRT stations and trains, and less so for buses.
-
-[Comment: You *really* don't want to commit even minor crimes here. Singapore
+- Drining and eating in public transport. You can be fined for it. This is especially true for MRT stations and trains.
```

Comment:
A missing `k` and also "drinking and eating" reads abit weird to me. Usually its "eating and drinking"? 😅 

```suggestion
- Eating and drinking in public transport. You can be fined for it. This is especially true for MRT stations and trains.
```